bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
I think `pry --gem` would offer such a console already? And `pry --gem --require /path/to/file.rb` would allow customizations?